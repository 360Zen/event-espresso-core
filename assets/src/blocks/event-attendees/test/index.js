/**
 * External dependencies
 */
import { blockEditRender } from '../../tests';

/**
 * Internal dependencies
 */
import { name, settings } from '../';

describe( name, () => {
	test( 'block edit matches snapshot', () => {
		const wrapper = blockEditRender( name, settings );
		expect( wrapper ).toMatchSnapshot();
	} );
} );
