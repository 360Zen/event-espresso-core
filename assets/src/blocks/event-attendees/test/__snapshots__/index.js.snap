// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`eventespresso/event-attendees block edit matches snapshot 1`] = `
<[object Object]
  value={
    Object {
      "clientId": undefined,
      "focusedElement": null,
      "isSelected": false,
      "name": "eventespresso/event-attendees",
      "setFocusedElement": [Function],
    }
  }
>
  <WithFilters(Edit)
    attributes={
      Object {
        "datetimeId": 0,
        "displayOnArchives": false,
        "eventId": 0,
        "limit": 10,
        "showGravatar": false,
        "status": "RAP",
        "ticketId": 0,
      }
    }
    isSelected={false}
    name="eventespresso/event-attendees"
    setAttributes={[Function]}
    user={Object {}}
  />
</[object Object]>
`;
